dwa_local_planner/src/dwa_planner_ros.cpp

```diff
@@ -263,28 +268,30 @@ namespace dwa_local_planner {
     //publish information to the visualizer
 
     publishLocalPlan(local_plan);
-    return true;
+    return mbf_msgs::ExePathResult::SUCCESS;
   }
 
-
-
-
-  bool DWAPlannerROS::computeVelocityCommands(geometry_msgs::Twist& cmd_vel) {
+  uint32_t DWAPlannerROS::computeVelocityCommands(const geometry_msgs::PoseStamped& pose,
+                                                  const geometry_msgs::TwistStamped& velocity,
+                                                  geometry_msgs::TwistStamped& cmd_vel, std::string& message) {
     // dispatches to either dwa sampling control or stop and rotate control, depending on whether we have been close enough to goal
     if ( ! costmap_ros_->getRobotPose(current_pose_)) {
-      ROS_ERROR("Could not get robot pose");
-      return false;
+      message = "Could not get robot pose";
+      ROS_ERROR_STREAM_NAMED("dwa_local_planner", message);
+      return mbf_msgs::ExePathResult::TF_ERROR;
     }
     std::vector<geometry_msgs::PoseStamped> transformed_plan;
     if ( ! planner_util_.getLocalPlan(current_pose_, transformed_plan)) {
-      ROS_ERROR("Could not get local plan");
-      return false;
+      message = "Could not get local plan";
+      ROS_ERROR_STREAM_NAMED("dwa_local_planner", message);
```

Comment:
S: create a static member for this `dwa_local_planner`

Reply:
oos, and anyway we won't use this for long